src/Sentry.Unity.Editor.Tests/Sentry.Unity.Editor.Tests.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutDir>$(PackageEditorTestsPath)</OutDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="UnityEngine">
+      <HintPath>$(UnityPath)\UnityEngine.dll</HintPath>
+      <Private>false</Private>
+    </Reference>
+    <Reference Include="UnityEditor">
+      <HintPath>$(UnityPath)\UnityEditor.dll</HintPath>
+      <Private>false</Private>
+    </Reference>
+    <Reference Include="UnityEditor.TestRunner">
+      <HintPath>$(UnityTestPath)\UnityEditor.TestRunner.dll</HintPath>
+      <Private>false</Private>
+    </Reference>
+    <Reference Include="UnityEngine.TestRunner">
+      <HintPath>$(UnityTestPath)\UnityEngine.TestRunner.dll</HintPath>
+      <Private>false</Private>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <!-- Unity uses exactly this (3.5.0) version as of now -->
+    <PackageReference Include="NUnit" Version="3.5.0" />

```

Comment:
We could control this version on the main `Directory.Build.Props`.

Or ideally like we do in .NET where the test projects are under `test` so there's a D.B.P there:

https://github.com/getsentry/sentry-dotnet/blob/main/test/Directory.Build.props

Reply:
Ok, I'll check!